_integration-test/ensure-customizations-work.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+set -ex
+
+source "$(dirname $0)/../install/_lib.sh"
+
+source ../install/dc-detect-version.sh
+
+# Negated version of ensure-customizations-not-present.sh, make changes in sync
```

Comment:
Almost certainly a way to DRY this up.

Reply:
I am sure there is, I am not convinced it is worth the effort though. I do follow DRY, but also WET - write everything twice. Given this is a negation, we will never need to maintain more than two versions of this.